frame/support/test/tests/pallet_ui/hooks_invalid_item.stderr

```diff
@@ -1,13 +1,13 @@
 error[E0107]: missing generics for trait `Hooks`
-   --> $DIR/hooks_invalid_item.rs:12:18
+   --> tests/pallet_ui/hooks_invalid_item.rs:12:18
```

Comment:
I am a little bit unsure about why I had to change this, but I assume it is harmless.